test/testdata/cert.pem

```diff
@@ -0,0 +1,15 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
What are these keys for?

Reply:
A server test use them: :)

https://github.com/kubernetes-sigs/cri-o/blob/71adf6b4735466caf5eb6a598732d19934f83c13/server/server_test.go#L52-L65